packages/edit-site/src/components/template-details/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export default function TemplateDetails( { template } ) {
+	if ( ! template ) {
+		return null;
+	}
+	const templateDescription = __(
+		'In the future, we will display more information about the template here.'
+	);
+
+	return (
+		<div className="edit-site-template-details">
+			<p className="edit-site-template-details__heading">
+				{ __( 'Template details' ) }
+			</p>
+			<p>{ `${ __( 'Name' ) }: ${ template.slug }` }</p>
```

Comment:
I wonder if we could reuse this, so we would have the same information here and in the navigation panel. This is not a blocker and it should be done in a separate PR since we should move this logic to an util or something.

Title: https://github.com/WordPress/gutenberg/blob/1bc9990a9f6383decb3323021d720ca59584e244/packages/edit-site/src/components/left-sidebar/navigation-panel/template-navigation-items.js#L42-L47

Description:
https://github.com/WordPress/gutenberg/blob/1bc9990a9f6383decb3323021d720ca59584e244/packages/edit-site/src/components/left-sidebar/navigation-panel/template-navigation-items.js#L49-L50

Reply:
Yeah, that's a great idea. I'll try to refactor that.